package.json

```diff
@@ -26,7 +26,7 @@
     "build": "gulp",
     "pretest": "gulp clean:tests",
     "test": "node --max-old-space-size=2048 node_modules/karma/bin/karma start",
-    "test:watch": "node --max-old-space-size=2048 node_modules/karma/bin/karma start --singleRun false",
+    "test:watch": "node --max-old-space-size=4096 node_modules/karma/bin/karma start --no-single-run",
```

Comment:
really necessary?

Reply:
No, I was testing some settings because I had a maximum call stack exceeded error in some UT. I forgot to remove it. Thanks